### ISSUE TITLE
when reload page the this.$auth.loggedIn change true. This time not login

### ISSUE THREAD

**Issue by sayi65** [nuxt-community/auth-module#355]
In Top page. the user not login.
but when reload page (F5) the $auth.loggedIn change to True.
is it bug?

#### Version
4.5.3

### nuxt.config.js
```javascript
 auth: {
    strategies: {
      local: {
        endpoints: {
          login: { url: '/api/auth/login', method: 'post', propertyName: 'data' },
          logout: false,
          user: false
        },
        tokenRequired: true,
        tokenType: 'Bearer'
      }
    },
    redirect: {
      login: '/',
      logout: '/'
    },
  },

```

<div align="right"><sub><em>This question is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/auth-module/issues/c336">#c336</a>)</em></sub></div>

**Comment by stact** [comment]
+1 same issue
Login 
`this.$auth.loginWith('local')` could navigate to private pages (loggedIn is `true`)

Logout
`this.$auth.logout()` redirect to home page (loggedIn is false), can't access to private pages (is redirecting to login page)

Go back to homepage and refresh with (F5) : private pages are available and loggedIn is `true`

**Comment by stact** [comment]
+ @pi0

**Comment by stact** [comment]
solved in 4.6.0 release
https://github.com/nuxt-community/auth-module/pull/346
